### PR TITLE
Update project structure

### PR DESCRIPTION
**Pull request by marcin-serwin**
This PR changes the license of the document to GPLv3, updates the title by dropping 2e and redesigns the cover page.

It is a WIP because I don't know how your friend would like to be credited for the image and under what license is it released (maybe [CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)?).

Also I was thinking maybe we could restructure the repository so it contains only the book? The other folders are not really related.

### CODE REVIEW

**Comment by oetiker** [comment]
The license for the Image is MIT
My friend is @bummzack

File: book/src/contrib.tex [hunk 1]
```diff
@@ -1,27 +1,71 @@
...
+  You should have received a copy of the GNU General Public License along with
+  this document.  If not, see \url{https://www.gnu.org/licenses/}.
+  \vspace{-5pt}
+  \begin{flushright}
+    \href{https://www.gnu.org/licenses/gpl-3.0.html}{%
+      \includegraphics[width=2cm]{images/gpl3.pdf}}\hspace*{10pt}
+  \end{flushright}
+
+  \bigskip\noindent Illustration on the first page---\enquote{ABC}, by DEF---is
+  licensed under a XYZ license.
```
Comment (oetiker) [inline book/src/contrib.tex:47]:
```suggestion
  \bigskip\noindent Illustration on the first page by Roman Schmid (@bumzack)---is
  dedicated to the public domain via CC0 1.0 Universal
  Public Domain Dedication.
```

**Review by oetiker — APPROVED** [review]

**Comment by oetiker** [comment]
I like the redesigned titlepage ... very nice

**Comment by marcin-serwin** [comment]
@oetiker What about this?
> Also I was thinking maybe we could restructure the repository so it contains only the book? The other folders are not really related.

Can I remove the other folders?

**Review by oetiker — APPROVED** [review]

**Pull request merged**